lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -184,6 +184,7 @@
       <display file="rviewer/bed.xml" inherit="true"/>
       <display file="igv/interval_as_bed.xml" inherit="true"/>
     </datatype>
+    <datatype extension="jf" type="galaxy.datatypes.binary:Binary" subclass="true" display_in_upload="true" description="Jellyfish database files are k-mer counts in binary format with a readable head. They are operated on and converted to human-readable text through jellyfish commands." />
```

Comment:
I would call the extension `jellyfish`. Two letters are really not that unique and meaningful.

Reply:
Good idea